jobs/github/scripts/goversion.sh

```diff
@@ -46,7 +46,14 @@ fi
 
 merge_commit=$(jq -r ".\"merge_commit_sha\"" "$tmpname")
 
-gomod=$(curl -s "https://raw.githubusercontent.com/$ghprbGhRepository/$merge_commit/go.mod")
+set +e
+gomod=$(curl -fs "https://raw.githubusercontent.com/$ghprbGhRepository/$merge_commit/go.mod")
+rval=$?
+if [ $rval -ne 0 ]; then
+  touch "${WORKSPACE}/goversion"
```

Comment:
does something always expect goversion to be there?

Reply:
yeah the builder expects it. Easier to just create it empty then writing a second set of checks when we already have one going.